src/views/Landing.vue

```diff
@@ -23,6 +23,12 @@
           principles to ensure our solutions are intuitive, performant, and
           deliver the best user experience.
         </p>
+        <a
```

Comment:
Can we make this into its own section? 
https://trello.com/c/rutLGtBv
ie it should be a copy of the "Mission Statement" page that only references our github.